pkg/hubble/parser/parser_test.go

```diff
@@ -29,7 +29,6 @@ import (
 	"github.com/cilium/cilium/pkg/monitor"
 	"github.com/cilium/cilium/pkg/monitor/api"
 	"github.com/cilium/cilium/pkg/proxy/accesslog"
-
```

Comment:
Is this removal intended?

Reply:
This was an unintentional change when I moved tests to `pkg/hubble/parser/agent`. Will revert, thanks.